packages/open-payments/src/client/requests.test.ts

```diff
@@ -365,4 +365,171 @@ describe('requests', (): void => {
       ).rejects.toThrow(/Failed to validate OpenApi response/)
     })
   })
+
+  describe('delete', (): void => {
+    const axiosInstance = createAxiosInstance({
+      requestTimeoutMs: 0,
+      privateKey,
+      keyId
+    })
+    const baseUrl = 'http://localhost:1000'
+    const responseValidators = mockOpenApiResponseValidators()
+
+    beforeAll(() => {
+      jest.spyOn(axiosInstance, 'delete')
+    })
+
+    test('properly makes DELETE request', async (): Promise<void> => {
+      const status = 202
+
+      // https://github.com/nock/nock/issues/2200#issuecomment-1280957462
+      jest
+        .useFakeTimers({
+          doNotFake: [
+            'nextTick',
+            'setImmediate',
+            'clearImmediate',
+            'setInterval',
+            'clearInterval',
+            'setTimeout',
+            'clearTimeout'
+          ]
+        })
+        .setSystemTime(new Date())
+
+      const scope = nock(baseUrl)
```

Comment:
```suggestion
      const scope = nock(baseUrl)
        .matchHeader('Signature', /sig1=:([a-zA-Z0-9+/]){86}==:/)
        .matchHeader(
          'Signature-Input',
          `sig1=("@method" "@target-uri");created=${Math.floor(
            Date.now() / 1000
          )};keyid="${keyId}";alg="ed25519"`
        )
```

Reply:
Is this necessary if there is no `Authorization` header? Do we want to update our interceptor to include `DELETE`s ?

Reply:
I guess not.
I think our only `DELETE`s will be for grants and tokens, both of which will have an authorization header access token.